fusedrug/data/tokenizer/modulartokenizer/modular_tokenizer.py

```diff
@@ -97,19 +91,14 @@ def __init__(
         self.tokenizers_info = ModularTokenizer.cfg_list_2_dict(
             copy.deepcopy(tokenizers_info_list)
         )
-        self.special_tokens_dict = special_tokens_dict
         self._max_possible_token_id = max_possible_token_id
         self._max_special_token_id = max_special_token_id
 
         if not load_adjusted_jsons:
             # store special tokens in a list to preserve their order:
-            all_special_tokens: List
-            if self.special_tokens_dict is None:
-                all_special_tokens = list([])
-            else:
-                all_special_tokens = list(self.special_tokens_dict.values())
-            if additional_tokens_list is not None:
-                all_special_tokens += additional_tokens_list
+            all_special_tokens = (
+                list(special_tokens_list) if special_tokens_list is not None else []
```

Comment:
don't we lose the scenario in which all_special_tokens was `list(self.special_tokens_dict.values())` and then `+= additional_tokens_list` ?

Reply:
oh, I see it's an argument that was removed from the signature